src/components/UpdateInput.tsx

```diff
@@ -1,14 +1,17 @@
+import { useRouter } from "next/router";
 import React from "react";
 import styled from "styled-components";
 
 import { Identity } from "../@types/Identity";
 import { useCookies } from "../hooks/useCookies";
+import { Button } from "../pages/account/logins/index";
```

Comment:
OK for now I've just duplicated the styled button in this file because later we'll need to create several buttons with different styles (add, cancel, delete...) 8113d96